contracts/weth-unwrapper/IWEthUnwrapper.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+interface IUnwrapper {
```

Comment:
I'd prefer if this had `weth` in the title, but I know a version is already on chain :/